src/apps/acb2wavs/acb2wavs.cpp

```diff
@@ -48,11 +48,11 @@ int main(int argc, const char *argv[]) {
 
 void PrintHelp() {
     cout << "Usage:\n" << endl;
-    cout << "acb2wavs <acb file> [-a <key1> -b <key2> -n]" << endl;
+    cout << "acb2wavs <acb file> [-a <key1> -b <key2>] [-n]" << endl;
     cout << "\t-n\tUse cue names for output waveforms]" << endl;
 }
 
-static char useCueName = 0;
+static bool useCueName = false;
```

Comment:
Use parameter/argument instead of global variable here. Generally, use global variables as little as possible.